File: codeforces/079/Cgen.cpp
#include <cstdio>
#include <cstdlib>

int main() {
	int n = 100000, k = 10;

	for (int i = 0; i < n; ++i) {
		putchar(rand() % 2 == 0 ? '0' : '1');
	}
	printf("\n%d\n", k);
	puts("010");
	puts("101");
	puts("000001");
	puts("100000");
	puts("100001");
	puts("0123456789");
	puts("9876543234");
	puts("1101110111");
	puts("1111111111");
	puts("0000000000");

	return 0;
}

File: fhc/2013r3/Cgen.rb
p 100
100.times do
  n = rand(1 .. 1111)
  k = rand(1 .. n)
  c = rand(1 .. 1000000000)
  puts "#{n} #{k} #{c}"
  puts Array.new(4){ rand(1 .. 100000000) } * ' '
  puts Array.new(4){ rand(1 .. 100000000) } * ' '
end
